client/scripts.js

```diff
@@ -1,15 +1,15 @@
 var socket = io({
-	reconnection: false
+	reconnection: false,
 });
 var rank = '';
 var name = '';
 var token = '';
 var rooms = {};
 var curRoom = '';
 var curMonth = '',
-	monthData = [];
+monthData = [];
 var curDay = '',
-	dayData = [];
+dayData = [];
```

Comment:
Notice the commas, the indentation is intentional here.

Reply:
oh